admin/acceptance_tests/layers_wmts_test.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=no-self-use
+
+import pytest
+
+from . import check_grid_headers
+
+
+@pytest.fixture(scope='class')
+@pytest.mark.usefixtures("dbsession")
+def insertLayerWMTSTestData(dbsession):
```

Comment:
question : can we change insertLayerWMTSTestData for insertTestData as we are in layers_wmts_test.py ?

Reply:
layer_wmts_test_data would be better dor me, as this is a fixture.
we need layer_wmts_test_data as dbsession or test_app.
note the example in layertree where I use `layer_wmts_test_data['themes']` in the tests